tests/float/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`float.rb 1`] = `
+6543.21
+6543.21.modulo(137.24)
+2.0**3
+-34.56.abs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+6543.21
+6543.21.modulo(137.24)
+2.0 ** 3
```

Comment:
Hey, there shouldn't be space around `**`.

Reply:
This is the default formatting for binary expressions from what I can see, so I think it's the right output here 🤔 

I can update the input file to have the whitespace around `**` so it's clearer though? 
Let me know your thoughts and I'll do whatever is needed :+1:

Reply:
Sorry misread it initially, this is good to go.